metricbeat/mb/registry.go

```diff
@@ -300,11 +300,33 @@ func (r *Register) Modules() []string {
 	r.lock.RLock()
 	defer r.lock.RUnlock()
 
+	var dups = map[string]int{}
```

Comment:
No need to keep a count, this could have a bool or an empty struct as value.

Reply:
I personally like empty `struct{}` because it's explicitly showing that the value doesn't matter.